dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -513,4 +513,43 @@ public void testUpdateUserMetadataFromGithub() {
         assertEquals("DockstoreTestUser2", userProfile.getUsername());
     }
 
+    @Test
+    public void testSetUserPrivilege() {
+        ApiClient adminWebClient = getWebClient(ADMIN_USERNAME, testingPostgres);
+        ApiClient userWebClient = getWebClient(USER_2_USERNAME, testingPostgres);
+
+        PrivilegeRequest privilegeRequest = new PrivilegeRequest();
+        UsersApi adminApi = new UsersApi(adminWebClient);
+        UsersApi userApi = new UsersApi(userWebClient);
+        User admin = adminApi.getUser();
+        User user = userApi.getUser();
+
+        privilegeRequest.setAdmin(false);
+        adminApi.setUserPrivilege(user.getId(), privilegeRequest);
+        assertFalse(userApi.getUser().isIsAdmin());
+        assertFalse(userApi.getUser().isCurator());
+
+        privilegeRequest.setCurator(true);
+        adminApi.setUserPrivilege(user.getId(), privilegeRequest);
+        assertTrue(userApi.getUser().isCurator());
+
+        try {
+            userApi.setUserPrivilege(admin.getId(), privilegeRequest);
+            assertFalse("Should have triggered an exception", false);
```

Comment:
1. Use `fail` instead of `assertFalse`.
2. Your error message should better indicate the problem to somebody looking at the code for the first time, e.g, `fail("Curator should not be able to create an admin")`

Reply:
Used fail() instead of assertFalse() and specified error message.